check.sh

```diff
@@ -120,6 +120,12 @@ if should_check enterprise_access; then
         "curl --fail -L http://localhost:18130/health/"
 fi
 
+if should_check enterprise_subsidy; then
```

Comment:
The subsidy service [container](https://github.com/edx/devstack/blob/a6e9652a2dcae89fdb0c3d2674206c470e21c328/docker-compose.yml#L915) is named as `enterprise-subsidy` so shouldn't the check be following instead? 
```suggestion
if should_check enterprise-subsidy; then
```